packages/connect-react/src/components/CorbadoConnectLoginContainer.tsx

```diff
@@ -23,6 +24,8 @@ const CorbadoConnectLoginContainer = () => {
         return <LoginErrorScreenSoft />;
       case LoginScreenType.ErrorHard:
         return <LoginErrorScreenHard />;
+      case LoginScreenType.Passkey:
```

Comment:
Maybe a more specific name makes sense here like "LoginScreenType.PasskeyReLogin" instead of "LoginScreenType.Passkey"? (a lot of the screens are about passkeys :))